### PR TITLE
[Security][SecurityBundle] Allow passing attributes to passport via `Security::login()`

### PR DESCRIPTION
**Pull request by SteNafH**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| License       | MIT

When using the `Security::login()` function it was not possible to set attributes on a passport. My authenticator is trying to get an attribute from a passport hence this PR.

### CODE REVIEW

File: src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php [hunk 1]
```diff
@@ -62,12 +62,17 @@ public function __construct(
     }
 
     /**
-     * @param BadgeInterface[] $badges Optionally, pass some Passport badges to use for the manual login
+     * @param BadgeInterface[]     $badges Optionally, pass some Passport badges to use for the manual login
+     * @param array<string, mixed> $attributes Optionally, pass some Passport attributes to use for the manual login
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php:66]: not sure why fabbot didn't pick this one
```suggestion
     * @param BadgeInterface[]     $badges     Optionally, pass some Passport badges to use for the manual login
     * @param array<string, mixed> $attributes Optionally, pass some Passport attributes to use for the manual login
```

**Review by nicolas-grekas — COMMENTED** [review]
This looks legit to me but please add tests.

**Comment by chalasr** [comment]
@SteNafH Do you plan to finish this PR anytime soon or would you like someone to take over?

**Comment by stefankleff** [comment]
Hi @chalasr - This isn't my PR :) It seems you tagged the wrong person

**Comment by SteNafH** [comment]
Hey @chalasr it is my pr ;), I have found another way of fixing the problem that made me open this PR in the first place. However I do feel that this is a good feature to have...

Sadly I don't have a lot of time on my hands and I would appreciate if someone else would takeover this pr!

**Comment by OskarStark** [comment]
@alexandre-daubois took over in
* #58166

Thanks for your work

**Pull request closed without merging**